Cargo.toml

```diff
@@ -65,19 +65,20 @@ name = "aws"
 [dependencies]
 anyhow = "1"
 async-trait = "0.1"
-base64 = "0.22"
+aws-credential-types = "1.1.8"
```

Comment:
I'm guessing this should not be in `dependencies`?

Reply:
Oops, done